docs/logging-and-managing-experiment-results/logging-experiment-data.rst

```diff
@@ -0,0 +1,1587 @@
+    You can create as many metrics as you wish.
+
+.. note::
+
+    Download metrics as pandas DataFrame for further analysis locally. Here is :ref:`how to do it <guides-download_data>`.
+
+:ref:`back to top <what-you-can-log>`
+
+.. _logging-experiment-data-parameters:
+
+Parameters
+^^^^^^^^^^
+Define parameters as Python dictionary and pass to the :meth:`~neptune.projects.Project.create_experiment` method to log them.
+
+.. code-block:: python3
+
+    # Define parameters
+    PARAMS = {'batch_size': 64,
+              'dense_units': 128,
+              'dropout': 0.2,
+              'learning_rate': 0.001,
+              'optimizer': 'Adam'}
+
+    # Pass parameters to create experiment
+    neptune.create_experiment(params=PARAMS)
+
++--------------------------------------------------------------------------------------------------------------+
+| .. image:: ../_static/images/logging-and-managing-experiment-results/logging-experiment-data/parameters.png  |
+|    :target: ../_static/images/logging-and-managing-experiment-results/logging-experiment-data/parameters.png |
+|    :alt: Experiment parameters                                                                               |
++==============================================================================================================+
+| Parameters in the experiment view                                                                            |
++--------------------------------------------------------------------------------------------------------------+
+
++------------------------------------------------------------------------------------------------------------------------+
+| .. image:: ../_static/images/logging-and-managing-experiment-results/logging-experiment-data/parameters-dashboard.png  |
+|    :target: ../_static/images/logging-and-managing-experiment-results/logging-experiment-data/parameters-dashboard.png |
+|    :alt: Experiment parameters in dashboard                                                                            |
++========================================================================================================================+
+| Parameters in the experiment dashboard                                                                                 |
++------------------------------------------------------------------------------------------------------------------------+
+
+You can use them later to analyse or compare experiments. They are displayed in the |parameters| section of the experiment. Moreover every parameter can be displayed as a column on the |experiment-dashboard| (look for green columns).
+
+|example-parameters|
+
+.. note::
+
+    Experiment parameters are read-only. You cannot change or update them during or after the experiment.
+
+:ref:`back to top <what-you-can-log>`
+
+.. _logging-experiment-data-code:
+
+Code
+^^^^
+Neptune supports code versioning. There are a few ways to do that.
+
+:ref:`back to top <what-you-can-log>`
```

Comment:
There is back to the top just a line above, I'd drop it

Reply:
ok